manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/ImagePullSecretHandler.java

```diff
@@ -64,9 +64,7 @@
         if (imageRegistry == null) {
             logger.debug("ImageRegistry is found to be null, skipping image pull secret creation ");
             String registry = serviceSpec.get(HyscaleSpecFields.getPath(HyscaleSpecFields.image, HyscaleSpecFields.registry), String.class);
-            if (StringUtils.isBlank(registry)) {
-                WorkflowLogger.persist(ManifestGeneratorActivity.FAILED_TO_CREATE_IMAGE_PULL_SECRET, registry, registry);
-            }
+            WorkflowLogger.persist(ManifestGeneratorActivity.FAILED_TO_CREATE_IMAGE_PULL_SECRET, "", "");
```

Comment:
Add registry = registry == null ? "" : registry.
And pass the registry in persist.